### DISCUSSION TITLE
Congrats! Your request for @RufatTech was approved and your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by RufatTech** [community/community#67790] [GitHub Education]
### Hi everyone,

2 days and no benefits

**Comment by priyanujgogoi-28** [comment]
In most cases, student benefits are generally given out to approved users in 24-72 hours (might be delayed by a day or two in rare occasions).

PS: For me, it took 3 days to receive the benefits. Also, it looks like you have received the benefits since I can see you have a Pro badge now.
